lms/djangoapps/instructor/views/gradebook_api.py

```diff
@@ -15,12 +17,33 @@
 from courseware.courses import get_course_with_access
 from instructor.offline_gradecalc import student_grades
 from instructor.views.api import require_level
-
+from instructor_task.models import ReportStore
 
 # Grade book: max students per page
 MAX_STUDENTS_PER_PAGE_GRADE_BOOK = 20
 
 
+def list_report_downloads_by_course_key(course_key):
+    """
+    Return json list of reports like grade book, ready for download
+
+    Arguments:
+        course_key (CourseKey): Course key
+
+    Returns:
+      string: Json list of reports ready for download.
+    """
+    report_store = ReportStore.from_config(config_name='GRADES_DOWNLOAD')
+
+    response_payload = {
+        'downloads': [
+            dict(name=name, url=url, link='<a href="{}">{}</a>'.format(url, name))
```

Comment:
Why do you need to construct this `link` parameter here? Why can't the client construct it, based on the name and URL?